### PR TITLE
feat: Restoring surge deployments for PR

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #

### CODE REVIEW

File: scripts/pr-deploy.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+    echo "Not a PR. Skipping surge deployment"
+    exit 0
+fi
+
+ember build production
```
Comment (iamareebjamal) [inline scripts/pr-deploy.sh]: This is already run

**Comment by kushthedude** [comment]
@iamareebjamal Surge deployments are working but the open-event-commenter app is not working which is used to comment the surge link on the PR

**Comment by iamareebjamal** [comment]
Use danger - https://danger.systems/

**Comment by kushthedude** [comment]
> Use danger - https://danger.systems/

Looking into it

File: scripts/pr-deploy.sh (hunk 1 above)
Comment (kushthedude) [reply scripts/pr-deploy.sh]: Ember Build Production is needed for succesfull surge deployment. I am thinking to put deploy preview in another job other than tests.

**Pull request closed without merging**